playbooks/installation/disconnected_satellite6.adoc

```diff
@@ -0,0 +1,141 @@
+= Satellite 6.1 Server Installation as an OSE3 Disconnected Registry
+
+== Purpose
+
+This document is an attempt to help consolidate and fill-in some of the existing documentation gaps for [Satellite 6.1] (https://access.redhat.com/documentation/en-US/Red_Hat_Satellite/6.1/pdf/Installation_Guide/Red_Hat_Satellite-6.1-Installation_Guide-en-US.pdf) and [Disconnected Docker Registries] (http://playbooks-rhtconsulting.rhcloud.com/playbooks/installation/disconnected_environments.html).  It is by no means a 100% complete solution, but rather answers a few FAQs and common frustrations while trying to install these products on the completlely Disconnected networks.
+
+== Assumptions
+
+You are installing Red Hat Satellite 6.1 on a Red Hat Enterprise Linux 7 Server with plenty (see requirements in Satellite 6.1 docs, above) of storage, CPU, RAM, and Internet access.  This host will then be removed from the Internet-connected network and connected to the disconnected network in order to serve the Red Hat Content (both RPMs and Docker Registry) to the disconnected network.
+
+TODO - Figure out and describe how this works with a katello-disconnected host.
```

Comment:
rather than putting a "TODO" here, can you add a 'Feedback Request' block per the style guide? [1]

[1] https://github.com/rhtconsulting/openshift-playbooks/blob/master/style_guide.adoc#requesting-feedback